src/Solver/time_stepping/TimeCluster.h

```diff
@@ -198,7 +198,7 @@ class seissol::time_stepping::TimeCluster
     double m_tv;
     
     //! Relax time for plasticity
-    double m_relaxTime;
+    double m_one_minus_integrating_factor;
```

Comment:
Seems like this line needs to be changed?
https://github.com/SeisSol/SeisSol/blob/faffbb2281dcd2fce3f7261dc903fe566f1dc4ec/src/Solver/time_stepping/TimeCluster.cpp#L640
How could it pass the compilations test?

Reply:
Could pass if no plasticity version is compiled (USE_PLASTICITY)

Reply:
Got the same error related to m_relaxTime, this could be related to this commit changing relaxTime to one_minus_integrating_factor:   https://github.com/SeisSol/SeisSol/commit/25dedbdef13079485f3b7bec37a58e0d144a28a2
testing the commit just before renaming works fine:
https://github.com/SeisSol/SeisSol/commit/693e2d327901acd5169cb691fc0e74f37d694ea8